xbmc/addons/interfaces/GUI/ListItem.cpp

```diff
@@ -66,7 +66,7 @@ void* Interface_GUIListItem::create(void* kodiBase, const char* label, const cha
   if (label2)
     item->get()->SetLabel2(label2);
   if (icon_image)
-    item->get()->SetIconImage(icon_image);
+    CLog::Log(LOGERROR, "ADDON::Interface_GUIListItem::%s - Using icon_image results in NOP and will soon be removed. Use setArt.", __FUNCTION__);
   if (path)
```

Comment:
Use a:
```cpp
item->get()->SetArt("icon", icon_image);
```
as long it is on `kodi::gui::CListItem` class constructor. 
Do not see it directly bad and would do for the purpose of removal there in a separate request.